class Test{
  void foo() throws java.io.<ref>IOException{}
}